core/chains/evm/client/errors.go

```diff
@@ -465,3 +465,22 @@ func NewSendErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction
 	}
 	return clienttypes.Unknown, err
 }
+
+// Solution to handle SendOnly nodes error codes. In that case, we don't assume there is another transaction that will be correctly
+// priced. Ideally in the future we wouldn't need an entire new function for this.
+func NewSendOnlyErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction, fromAddress common.Address, isL2 bool) (clienttypes.SendTxReturnCode, error) {
```

Comment:
Usually the linter insists on exported API being documented by staring with its own name:
```suggestion
// NewSendOnlyErrorReturnCode handles SendOnly nodes error codes. In that case, we don't assume there is another transaction that will be correctly
// priced. Ideally in the future we wouldn't need an entire new function for this.
func NewSendOnlyErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction, fromAddress common.Address, isL2 bool) (clienttypes.SendTxReturnCode, error) {
```